draft-ietf-taps-interface.md

```diff
@@ -1812,30 +1829,29 @@ Type:
 Default:
 : Disabled
 
-This property specifies how long to wait before deciding that an active Connection has
+This property specifies how many seconds to wait before deciding that an active Connection has
 failed when trying to reliably deliver data to the Remote Endpoint. Adjusting this Property
 will only take effect when the underlying stack supports reliability. The special value
-`Disabled` means that this timeout is not scheduled to happen.
+`Disabled` means that no timeout is scheduled.
 
 ### Timeout for keep alive packets {#keep-alive-timeout}
 
 Name:
 : keepaliveTimeout
 
 Type:
-: Numeric, with special value `Default`
```

Comment:
I removed `Default` here because of the new `Unset` method. Does there need to be a `Disabled` value for when keepalive functionality is off?

Reply:
Resolve in accordance with removing Unset

Reply:
Not sure if I got this right. Check the corresponding text in the updated PR.